File: Aula07_ExemploMVCSinglePageApp/Exemplo_Objc/ViewController.m
//
//  ViewController.m
//  Exemplo_Objc
//
//  Created by Usuário Convidado on 26/03/2018.
//  Copyright © 2018 Usuário Convidado. All rights reserved.
//

#import "ViewController.h"
#import "Pessoa.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)calcular:(id)sender {
    
    Pessoa *p = [[Pessoa alloc]init];
    
    p.peso = self.txtPeso.text.floatValue;
    p.altura = self.txtAltura.text.floatValue;
    p.nome = self.txtPeso.text;
    
    self.txtIMC.text = [NSString stringWithFormat:@"%0.2f",[p calcularIMC_2]];
}


@end

File: Swift/Exemplo Swift/Exemplo Swift/ViewController.swift
//
//  ViewController.swift
//  Exemplo Swift
//
//  Created by Usuário Convidado on 09/04/2018.
//  Copyright © 2018 Joao Victor Ferreira Lima. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var txtNome: UITextField!
    @IBOutlet weak var txtPeso: UITextField!
    @IBOutlet weak var txtAltura: UITextField!
    @IBOutlet weak var txtIMC: UITextField!
    
    @IBAction func calcular(_ sender: Any) {
        let p = Pessoa()
        p.nome = txtNome.text
        p.peso = Float(txtPeso.text!)
        p.altura = Float(txtAltura.text!)
        p.calcularImc()
        txtIMC.text = "\(p.imc!)"
    }
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

